.travis.yml

```diff
@@ -12,7 +12,7 @@ language: node_js
 if: type != push OR (tag IS blank AND branch = master)
 
 install:
-  - yarn
+  - yarn install --ignore-engines
```

Comment:
Why is this on by default in yarn?  And if it's a good reason, why don't we fix the problems we're having with it?

Reply:
because yarn is weird with engines. not sure how to fix it